js/main.js

```diff
@@ -103,7 +103,7 @@ requirejs.config({
 		"negative-controls": "components/negative-controls",
 		"d3": "d3.min",
 		"d3_tip": "d3.tip",
-		"jnj_chart": "jnj.chart",
+		"jnj_chart": "https://cdn.rawgit.com/odysseusinc/atlascharts/master/index",
```

Comment:
@t-abdul-basser, @chrisknoll, so how can we solve the issue with module's repo? Will it be possible for someone,  having permissions,  to create an appropriate repo in OHDSI space? Then we can put module there and modify this Atlas'es pull-request